azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/CallCompositeActivity.kt

```diff
@@ -153,7 +153,7 @@ internal class CallCompositeActivity : AppCompatActivity() {
                 supportedOSLocale()
             }
             else -> {
-                configuration.localizationConfig!!.layoutDirection.let {
+                configuration.localizationConfig!!.layoutDirection?.let {
```

Comment:
is layoutDirection nullable?

Reply:
No. As we are making it final and assigned value where we are not taking the direction from parameter. removing the nullable operator.